budget/app.py

```diff
@@ -8,38 +10,64 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-# TODO configure this URL?
-SYNAPSE_TEAM_MEMBER_URL = 'https://repo-prod.prod.sagebase.org/repo/v1/teamMembers'
-
 # TODO pull this config from S3?
-USER_BUDGET_BY_TEAM = {
-  '12345': {
-    'amount': 2000,
-    'period': 'annual'
+# and validate it with jsonschema in a future PR
+# this configuration will differ between accounts
+
+TEAM_BUDGET_RULES = {
+  'teams': {
+    '3412821': {
+      #'amount': '2000',
+      'amount': '10',
+      'period': 'ANNUALLY',
+      'community_manager_emails': ['tess.thyer@sagebase.org']
+    }
   },
-  '67890': {
-    'amount': 100,
-    'period': 'annual'
+  'thresholds': {
+    'notify_user_only': [25.0, 50.0, 80.0],
+    'notify_admins_too': [90.0, 100.0, 110.0]
```

Comment:
Good idea! Would the logic notify on all values >110%? For example, if I went from 89% to 112% of my budget, would an admin be notified?

Reply:
yes